client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -469,24 +470,40 @@ function PurchaseMetaExpiration( {
 		const subsRenewText = isAutorenewalEnabled
 			? translate( 'Auto-renew is ON' )
 			: translate( 'Auto-renew is OFF' );
-		const subsBillingText =
-			isAutorenewalEnabled && ! hideAutoRenew && hasPaymentMethod( purchase )
-				? translate( 'You will be billed on {{dateSpan}}%(renewDate)s{{/dateSpan}}', {
-						args: {
-							renewDate: purchase.renewDate && moment( purchase.renewDate ).format( 'LL' ),
-						},
-						components: {
-							dateSpan,
-						},
-				  } )
-				: translate( 'Expires on {{dateSpan}}%(expireDate)s{{/dateSpan}}', {
-						args: {
-							expireDate: moment( purchase.expiryDate ).format( 'LL' ),
-						},
-						components: {
-							dateSpan,
-						},
-				  } );
+		let subsBillingText;
+		if (
+			isAutorenewalEnabled &&
+			! hideAutoRenew &&
+			hasPaymentMethod( purchase ) &&
+			isRechargeable( purchase )
+		) {
+			subsBillingText = translate( 'You will be billed on {{dateSpan}}%(renewDate)s{{/dateSpan}}', {
+				args: {
+					renewDate: purchase.renewDate && moment( purchase.renewDate ).format( 'LL' ),
+				},
+				components: {
+					dateSpan,
+				},
+			} );
+		} else if ( ! isRechargeable( purchase ) && hasPaymentMethod( purchase ) ) {
+			subsBillingText = translate( 'Expires on {{dateSpan}}%(expireDate)s{{/dateSpan}}', {
```

Comment:
Unless I'm missing something... this block does the same thing as the `else` below, right? If so, can we remove this one?

Reply:
Thanks for catching this one. I did some manual testing and have removed the else if block.